RpcRenderMeshBuilder.cpp

```diff
@@ -437,15 +437,14 @@ bool CRpcRenderMeshBuilder::OldTexture2Material(RPCapi::Texture& RpcTexture, RPC
 		{
 			for (int x = 0; x < iWidth; x++)
 			{
-				rDib.SetPixel(x, y, alp[0], alp[0], alp[0], 255);
+				rDib->SetPixel(x, y, alp[0], alp[0], alp[0], 255);
 				alp += 1;
 			}
 		}
 	}
 
-	// TODO: [HERE] Possible dib ownership problem - check RDK SDK comments.
-	CRhRdkTexture* pRdkTexture = RhRdkNewDibTexture(&rDib, Material.DocumentAssoc(), false, true);
-
+	CRhRdkTexture* pRdkTexture = RhRdkNewDibTexture(rDib, Material.DocumentAssoc());
+    dibs.Append(rDib);
```

Comment:
If the original intention was to hang onto the DIBs, so you solve the ownership issue, that's no longer necessary with the shared_ptr.